lib/segment/src/vector_storage/quantized/quantized_vectors.rs

```diff
@@ -691,13 +867,40 @@ impl QuantizedVectors {
         Ok(quantized_vectors)
     }
 
-    pub fn config_exists(path: &Path) -> bool {
-        path.join(QUANTIZED_CONFIG_PATH).exists()
+    pub fn load(
```

Comment:
How about we call this `load_or_create`?

One of the main things I find very confusing in our current storage implementations is all the inconsistencies in what does or does not create new data if it doesn't exist.